src/katsdpcontroller/schemas/product_config.json.j2

```diff
@@ -585,6 +588,24 @@
                                 "additionalProperties": false,
                                 "required": ["src_streams", "src_pol"]
                             }
+                        },
+                        {
+                            "if": {"properties": {"type": {"const": "gpucbf.tied_array_resampled_voltage"}}},
+                            "then": {
+                                "properties": {
+                                    "type": {},
+                                    "src_streams": {"$ref": "#/definitions/pair"},
+                                    "n_chans": {"$ref": "#/definitions/positive_integer"},
+                                    "pols": {"$ref": "#/definitions/pair"},
```

Comment:
The spec is more specific: it has to be either ["x", "y"] or "["R", "L"]. I think the "enum" JSONSchema type is suitable for this: something like

```suggestion
                                    "pols": {"enum": [["x", "y"], ["R", L"]]},
```